l10n_ec_delivery_note/models/sale_order.py

```diff
@@ -0,0 +1,44 @@
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
```

Comment:
Maybe you can keep this feature on another module, with auto_install True, to allow users install account, stock without sales